lib/workers/repository/update/branch/auto-replace.spec.ts

```diff
@@ -182,4 +182,706 @@ describe('workers/repository/update/branch/auto-replace', () => {
       await expect(res).rejects.toThrow(WORKER_FILE_UPDATE_FAILED);
     });
   });
+
+  describe('doReplacementAutoReplace', () => {
+    let reuseExistingBranch: boolean;
+    let upgrade: BranchUpgradeConfig;
+
+    beforeEach(() => {
+      upgrade = {
+        ...JSON.parse(JSON.stringify(defaultConfig)),
+      };
```

Comment:
```suggestion
      upgrade = getConfig();
```

Reply:
The config used in this file before my changes is also created via json.parse.
getConfig returns an AllConfig object as opposed to the BranchUpgradeConfig used in the doAutoReplace function.
Multiple properties used are not part of AllConfig.

Reply:
see updated suggestions